ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -506,13 +506,15 @@ public void shouldReturn5xxOnStatementSystemError() {
     final String ksqlString = "CREATE STREAM test_explain AS SELECT * FROM test_stream;";
     givenMockEngine(mockEngine -> {
       EasyMock.expect(mockEngine.parseStatements(EasyMock.anyString()))
-          .andReturn(realEngine.parseStatements(ksqlString));
-
+          .andDelegateTo(realEngine);
+      EasyMock.expect(
```

Comment:
If you switch to Mockito you won't need to add this. The default would be to return 0.

Reply:
Thoughts? It's only suggestion: it's OK to say you'd rather not / don't have time / next time / separate PR.